lib/xcpretty/formatters/simple.rb

```diff
@@ -77,6 +79,12 @@ def format_pending_test(suite, test_case)
       INDENT + format_test("#{test_case} [PENDING]", :pending)
     end
 
+    def format_measuring_test(suite, test_case, time)
```

Comment:
Unused method argument - `suite`. If it's necessary, use `_` or `_suite` as an argument name to indicate that it won't be used.